### ISSUE TITLE
"nullable" is missing in response schema on primitive nullable type

### ISSUE THREAD

**Issue by skironDotNet** [domaindrivendev/Swashbuckle.AspNetCore#2731]
Simple endpoint

        [ApiVersion("1.0")]
        [Route("intnull")]
        [ProducesResponseType(StatusCodes.Status200OK)]
        [HttpGet]
        [SwaggerOperation(OperationId = "IntNull")]
        [SwaggerResponse(200, "nullable int", typeof(Nullable<int>))]  //this makes no diffrence
        public async Task<ActionResult<Nullable<int>>> IntNull([FromQuery] Dictionary<string, DateTime> test)
        {
            return this.Ok(null);
        }
  
the output
```
  "responses": {
          "200": {
            "description": "nullable int",
            "content": {
              "text/plain": {
                "schema": {
                  "type": "integer",
                  "format": "int32"
                }
              },
              "application/json": {
                "schema": {
                  "type": "integer",
                  "format": "int32"
                }
              },
              "text/json": {
                "schema": {
                  "type": "integer",
                  "format": "int32"
                }
              }
            }
```

The schema should be
```
         "schema": {
                  "type": "integer",
                  "format": "int32",
                  "nullable": true
                }
```

but "nullable" is missing

**Comment by Havunen** [comment]
This issue is fixed in DotSwashbuckle 3.0.2
https://github.com/Havunen/DotSwashbuckle
https://www.nuget.org/packages/DotSwashbuckle.AspNetCore

**Comment by martincostello** [comment]
To make issue tracking a bit less overwhelming for the new maintainers (see #2778), I've created a new tracking issue to roll-up various nullability issues here: #2793.

We'll refer back to this issue from there and include it as part of resolving that issue, but I'm going to close this one to help prune the backlog.
